Dockerfile.ppc64le

```diff
@@ -0,0 +1,19 @@
+FROM mambaorg/micromamba
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+USER root
+
+RUN apt-get update  -y     && apt-get install -y git wget vim numactl gcc-12 g++-12 protobuf-compiler libprotobuf-dev     && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-12 10 --slave /usr/bin/g++ g++ /usr/bin/g++-12
+
+RUN micromamba install -y -n base -c rocketce  -c defaults     python=3.10     pytorch-cpu     numba     llvmlite     httptools &&     micromamba clean --all --yes
```

Comment:
Can you add a note that pytorch is installed here, rather than in the requirements-cpu.txt. How is the versioning controlled? It seems to be unrestricted here atm

Reply:
Right now the available packages are limited for PowerPC (aka pytorch and triton).  Until the versions required in `requirements-cpu.txt` are met for PPC, this might be the best that can be done.